ntp-proto/src/time_types.rs

```diff
@@ -179,12 +179,34 @@ impl NtpDuration {
         Self { duration }
     }
 
+    pub(crate) fn from_exponent(input: i8) -> Self {
+        Self::from_seconds(log2d(input))
+    }
+
+    #[must_use]
+    pub(crate) const fn divided_by(self, divisor: i64) -> Self {
+        // No overflow risks for division
+        Self {
+            duration: self.duration / divisor,
+        }
+    }
```

Comment:
I needed const division.

Reply:
I guess this is not fixable by adding const to the div implementation?

Reply:
I guess this is not fixable by adding const to the div implementation?

Reply:
sadly not, traits cannot have const methods yet.